### DISCUSSION TITLE
Defining a detection area for the Barcode Scanner

### DISCUSSION THREAD

**Discussion by rveltonCL** [capawesome-team/capacitor-mlkit#64] [General]
In the help docs, it mentions something about defining a detection area? Does anyone have a sample on how to get this done? Thanks.

**Comment by robingenz** [comment]
Hey @rveltonCL,
Here you can find the official demo: https://github.com/robingenz/capacitor-mlkit-plugin-demo/blob/main/src/app/modules/barcode-scanning/barcode-scanning-modal.component.ts
